.github/workflows/deploy_docs.yml

```diff
@@ -58,7 +58,6 @@ jobs:
         env:
           GOOGLE_CALENDAR_ID: ${{ secrets.GOOGLE_CALENDAR_ID }}
           GOOGLE_CALENDAR_API_KEY: ${{ secrets.GOOGLE_CALENDAR_API_KEY }}
-          PIP_CONSTRAINT: ${{ github.workspace }}/napari-repo/resources/constraints/constraints_py3.10_docs.txt
```

Comment:
we need this env var because `make docs` does a pip install.

Reply:
:cry: Its so confusing #291

Reply:
Ah sorry I think I deleted this on error due to rebasing my local `main`. I'll fix before marking ready to review.